ReflectionTest/ReflectionConsoleTest/Program.cs

```diff
@@ -12,62 +12,61 @@ class Program
     {
         static void Main(string[] args)
         {
-            try
+            var user = new User("tsar", "12345", true);
+            var users = new UserStorage();
+
+            for (int i = 0; i < 20; i++)
             {
-                var user = new User("tsar", "12345", true);
-                var users = new UserStorage();
+                users.AddRandomUser();
+            }
 
-                for (int i = 0; i < 20; i++)
-                {
-                    users.AddRandomUser();
-                }
+            users.AddUser(user);
+            users.DeleteUser(users.FindByLogin("tsar"));
+            users.DeleteUser(user);
 
-                users.AddUser(user);
-                users.DeleteUser(users.FindByLogin("tsar"));
-                users.DeleteUser(user);
-                users.DeleteUser(user);
+            GetReflectionInfo("SomeLibrary");
 
-                
-                var assemblies = AppDomain.CurrentDomain.GetAssemblies();
-                Type[] typelist = new Type[0];
-                foreach (var assembly in assemblies)
-                {
-                    typelist = assembly.GetTypes().Where(t => t.Namespace == "SomeLibrary").ToArray();
+            Console.WriteLine("Press any key...");
+            Console.ReadKey();
+        }
 
-                    if (typelist.Any())
-                    {
-                        break;
-                    }
-                }
+        public static bool GetReflectionInfo(string assemblyNamespace)
+        {
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
+            IEnumerable<Type> types = new Type[0];
```

Comment:
new Type[0]; is created and trown away after variable is assigned is Linq enumerable.

Reply:
I need to initialize "types", so I assigned a null.